sw/device/lib/crypto/include/aes.h

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_CRYPTO_INCLUDE_AES_H_
+#define OPENTITAN_SW_DEVICE_LIB_CRYPTO_INCLUDE_AES_H_
+
+/**
+ * @brief AES operations for the OpenTitan cryptography library.
```

Comment:
Taking the DIFs as a reference, should you put a `@file` tag here as well?

Reply:
Done!